client/components/StickyNotes.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useRef } from 'react';
+import logoBw from '../../server/public/img/logo_bw.png';
+import { makeWidgetMovable } from '../functions/widgetMove';
+
+const StickyNotes = () => {
+  const widgetRef = useRef();
+
+  const handleWidgetMovement = makeWidgetMovable(widgetRef);
```

Comment:
This is great. If you want to improve it, you could make a "MovableWidget" component that accepts a "children" prop. It would handle calling `useRef`, the `handleWidgetMovement` function, and so on.

If I were you, I'd push forward with what you've got since it works and isn't that bad. Then when you have time for clean up you could look into this approach.